projects/gnu-tools/additional-materials/supplementary-notes.md

```diff
@@ -0,0 +1,25 @@
+# CORE-V Tool Chain supplementary notes
+
+## Repositories
+
+Binutils-gdb - https://github.com/openhwgroup/corev-binutils-gdb
+GCC - https://github.com/openhwgroup/corev-gcc
+CORE-V docs - https://github.com/openhwgroup/core-v-sw
```

Comment:
As Markdown these fold into a single line. Easiest solution is probably to turn them into a list.
```
- Binutils-gdb - https://...
- GCC - ...
- etc
```